source/PrintData.cpp

```diff
@@ -164,7 +164,7 @@ namespace {
 				cout << ship.Cost() << ',';
 
 				auto mass = attributes.Mass() ? attributes.Mass() : 1.;
-				cout << attributes.Get("shields") << ',';
+				cout << ship.MaxShields() << ',';
 				cout << attributes.Get("hull") << ',';
```

Comment:
Why are we using MaxShields above this but not using MaxHull?
Line 165 of DamageProfile would also need this.